tests/test_codex_feed_hooks.py

```
@@ -528,6 +601,67 @@ def test_install_adds_codex_permission_request_hook(cli_path: str, root: Path) -
         raise AssertionError(f"hooks feature was not enabled: {config_toml!r}")
     if "codex_hooks" in config_toml:
         raise AssertionError(f"deprecated codex_hooks feature was written: {config_toml!r}")
+    config = tomllib.loads(config_toml)
+    state = config.get("hooks", {}).get("state", {})
+    expected_trust = expected_cmux_codex_hook_trust(hooks, codex_home / "hooks.json")
+    if len(expected_trust) != 5:
+        raise AssertionError(f"expected 5 cmux Codex trust entries, got {expected_trust!r}")
```

Ruff (0.15.12)

[warning] Avoid specifying long messages outside the exception class

(TRY003)